### ISSUE TITLE
Support for GMT (Zulu) time

### ISSUE THREAD

**Issue by psztoch** [belgianbeer/tzdiff#6]
`date` command support time zone offset:
2022-03-26T21:00Z
2022-03-26T21:00+01
2022-03-26T21:00+03

It is compatible format with ISO 8601.

```
date -d 2022-03-26T21:00Z +%s
date -d 2022-03-26T21:00+00 +%s
date -d 2022-03-26T21:00+02 +%s
date -d 2022-03-26T21:00+04 +%s
```

Accept it as -t argument.

**Comment by belgianbeer** [comment]
It seems that the date command on your system is GNU date. Many Linux systems have GNU date, and the -d option of GNU date is very flexible in interpreting time strings. 

However, the date command for macOS, FreeBSD, OpenBSD, etc. is not a GNU one and has no equivalent to the -d option. I am not going to assume GNU date since I mainly use FreeBSD.

I think I can make it so that if Z is at the end of the -t string, it means UTC. I will implement this in the next version.

**Comment by belgianbeer** [comment]
I'm updating tzdiff now.

The "work" branch of tzdiff has been updated. If your system has the GNU date command, you can specify the time in the format you want.

**Comment by belgianbeer** [comment]
Released tzdiff 1.2 with the ability to specify times in UTC.
Thanks for your request.

**Comment by belgianbeer** [comment]
Released tzdiff 1.2 with the ability to specify times in UTC.
Thanks for your request.
